examples/generic_connector_configs/README.md

```diff
@@ -1,28 +1,221 @@
 # Generic HTTP Connector Example Configurations
 
+## Table of Contents
+- [Generic HTTP Connector Example Configurations](#generic-http-connector-example-configurations)
+  * [Table of Contents](#table-of-contents)
```

Comment:
```suggestion
```
I think it's weird when the title of the page and the TOC are referenced in the TOC